README.md

```diff
@@ -197,6 +197,16 @@ const App = () => (
 );
 ```
 
+#### Restore default browser navigation for one link
+
+The router intercepts `<a />` links that point to app pages. When you click such a link, it switches the route without loading a full page. It's quite common for users to want to opt-out of this behavior. Here are the rules for making `preact-iso` ignore a link:
+
+1. You're on `https://example.org` but the link points to `https://example.com`.
+2. The link points to an identifier on the current page, i.e. to `#section`.
```

Comment:
"fragment" is another keyword that's just useful to have around, even if it doesn't really add much to the sentence, and I figure a more explicit example similar to what you have above re:cross-origin links couldn't hurt.

```suggestion
2. The link points to a fragment identifier on the current page, i.e., you're on `/foo` and the link points to `/foo#section`.
```